server/routes/activityRouter.js

```diff
@@ -0,0 +1,87 @@
+var Activity = require('../models/activity');
+var express = require('express');
+
+const activityRouter = express.Router();
+
+activityRouter.route('/')
+    .get((req, res) => {
+        Activity.find({}, (err, activities) => {
+            res.json(activities)
+        })
+    })
+
+activityRouter.route('/:householdId/:studentId')
```

Comment:
Add in comments to your routes to explain what the purpose of them are so we'll be able to quickly understand them in the future!

/**
* This route takes in a household id and student id and does xyz
*/